### ISSUE TITLE
Controls not trigger when change IME(Keyboard layout)

### ISSUE THREAD

**Issue by 312784123** [bluemagic123/tModLoader#3095]
### Version

Latest 1.4-stable

### OS

Windows

### Platform

Steam

### This bug affects

Gameplay as a Player

### Description

![control](https://user-images.githubusercontent.com/118354239/202164736-934e6f7e-2ea3-4cb0-9825-a252a54044ea.gif)

Controls isn't triggerd when changing IME(i thought. because similar issue happened minecraft. In minecraft case, issue was solved by IME modding,  https://github.com/Axeryok/CocoaInput . but I'm not sure that issues are equal)

Maybe, similar issue happened with cyrillic (https://github.com/tModLoader/tModLoader/issues/2117)

but Korean control issue happened after August release not January

https://github.com/tModLoader/tModLoader/releases/tag/v2022.06.96.6

issue doesn't happen this version

https://github.com/tModLoader/tModLoader/releases/tag/v2022.07.58.1

but happen after this version

I would appreciate it if you could solve this problem or let me know how to solve this problem

Thanks for developing Tmodloader

### Steps to reproduce

Change IME(maybe) if don't have Korean
then, Add Window Language Pack(don't have to set default Korean just add)
after that, Window task bar has IME

![캡처](https://user-images.githubusercontent.com/118354239/202171348-022cf2b4-5489-4476-a92d-6bd4db94e82d.PNG)

like this

toggle Korean

now, in tmodloader, Not triggering control

### Expected Behavior

Player can't trigger control(Especially Letter part)

### Reproduction frequency

always when toggle Korean

**Comment by Cyrillya** [comment]
This happens to Chinese IME as well #2571 and yeah you can't trigger controls after August release

**Comment by 312784123** [comment]
> This happens to Chinese IME as well #2571 and yeah you can't trigger controls after August release


I tested some cases in 7.58.1 and 9.47.14 version, Korean(after August) not trigger , Japanese(I don't know when issue happen) not trigger, Chinese(#2571 , after June) not trigger, Russian-Cyrillic( #2117 after January, But In my test, I used Russian, Russian-Ukrainian, Cyrllic-serbia. all triggers control. In these cases. I don't use IME because Microsoft did not offer IME)

so Using CJK and etc. IME in tmodloader looks problematic, but What makes difference in when to happen IME issue?
Why didn't issue happen at the same time?
